autogold.go

```diff
@@ -19,8 +19,10 @@ var (
 	update       = flag.Bool("update", false, "update .golden files, removing unused if running all tests")
 	updateOnly   = flag.Bool("update-only", false, "update .golden files, leaving unused")
 	noUpdateFail = flag.Bool("no-update-fail", false, "do not fail tests if .golden file was updated")
-	cleaned      = map[string]struct{}{}
-	cleanDir     string
+
+	cleanLock sync.Mutex
```

Comment:
nit: traditionally mutex hats will be named to match the first var it protects (`cleanLock` would be `cleanedMu`): https://dmitri.shuralyov.com/idiomatic-go#mutex-hat

Reply:
Apologies, I mean to fix that naming before pushing the change. I’ll fix it up

Reply:
Fixed